config/initializers/simple_form_bootstrap.rb

```diff
@@ -1,10 +1,8 @@
 # frozen_string_literal: true
 
-# Please do not make direct changes to this file!
-# This generator is maintained by the community around simple_form-bootstrap:
+# These defaults were defined by the community at
```

Comment:
I think it's nice to say it's maintained as well, so it's clear they can look to that for updates, wdyt?

Maybe something like:

```suggestion
# These defaults are defined and maintained by the community at
```